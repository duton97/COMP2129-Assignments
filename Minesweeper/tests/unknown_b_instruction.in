g 3 4
q 0 0
