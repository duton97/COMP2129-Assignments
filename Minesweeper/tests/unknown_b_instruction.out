g 3 4
error
